scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -155,7 +155,7 @@ <h3 class="subnav__page-title">
                     ></div>
                 </div>
                 <div ng-if="monitoring.singleGroup && !monitoring.isDeskChanged()" class="list">
-                    <div sd-monitoring-group class="single-group" data-group="monitoring.singleGroup" data-num-items="10" data-view-type="'single_monitoring'">
+                    <div sd-monitoring-group class="single-group" data-group="monitoring.singleGroup" data-num-items="50" data-view-type="'single_monitoring'">
```

Comment:
@tolga, seems like num-items not utilized in directive code. imo you can remove this for this PR. and that part 
- Changed the initial number of items in single group from 10 to 50.

I will handle it in separate PR.
Rest of changes LGTM.